tests/abortStatus/abortStatus.edl

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+enclave {
+    trusted {
+        public int regular_abort();
+
+        public int generate_unhandled_hardware_exception();
+
+        public int test_ocall_after_abort(
+            [in] void** thread_ready_count,
```

Comment:
It seems more natural to pass this as
[user_check] void* thread_ready_count
[user_check] void* is_enclave_crashed